app/views/mobileValidate.scala.html

```diff
@@ -0,0 +1,274 @@
+@import models.user.User
+@import models.mission.{MissionTable, Mission}
+@import models.label.LabelTable
+@import models.label.LabelTable.LabelValidationMetadata
+@import play.api.libs.json.Json
+@import play.api.libs.json.JsValue
+@import views.html.bootstrap._
+
+@(title: String, user: Option[User] = None, mission: Option[JsValue], labelList: Option[JsValue], progress: Option[JsValue], hasNextMission: Boolean)
+
+@main(title) {
+    <script type="text/javascript" src='@routes.Assets.at("javascripts/SVValidate/build/SVValidate.js")'></script>
+    <script type="text/javascript" src='@routes.Assets.at("javascripts/SVLabel/lib/gsv/zpipe.min.js")'></script>
+    <script type="text/javascript" src='@routes.Assets.at("javascripts/SVLabel/lib/gsv/jquery.base64.min.js")'></script>
+    <script type="text/javascript" src='@routes.Assets.at("javascripts/SVLabel/lib/kinetic-v4.3.3.min.js")'></script>
+    <link rel="stylesheet" href='@routes.Assets.at("javascripts/SVValidate/build/SVValidate.css")'/>
+    <link rel="stylesheet" href='@routes.Assets.at("javascripts/SVValidate/mobile/mobileValidate.css")'/>
+
+    <script>
+            $(document).ready(function() {
+
+                //raleway doesn't load after being redirected from another page; redeclaring the font-face fixes that
```

Comment:
I think you didn't finish what you were commenting here?

Reply:
This comment makes sense to me? Ill add another for the orientationCheck()

Reply:
oops you're right, I misread it the first time :)